lib/nexmo_developer/app/assets/javascripts/digitalData.js

```diff
@@ -1,11 +1,13 @@
+const PRIMARY_CATEGORY = document.getElementById('adobe_launch_primary_category').value
```

Comment:
Can we use environment variables in JavaScript rather than going through ruby & HTML?

Reply:
Nick and I decided to move this to production since we cannot read PRIMARY_CATEGORY directly from the environment variable. The dotenv(https://zetcode.com/javascript/dotenv/) implementation suggested by Paul works only through node.js as it is a server-side javascript. We are using a client-side javascript. I would like to have an understanding on this before I start working on Phase2 of Adobe Analytics so that we know why we need this change and will also be useful for my further development.